bash_command_line_101/bash_command_line_101.md

```diff
@@ -480,17 +480,15 @@ The quotes around the text can also be safely omitted in this case, but that isn
 ***
 
 
-## Additional Resources
+## Getting Help from the Manual
 
-- [Brief Illustration of the Difference between Shell and Kernel](https://www.geeksforgeeks.org/difference-between-shell-and-kernel/)
+<div class="warning">
 
-- [Reinforce Your New Knowledge through this Learning the Shell Page](https://linuxcommand.org/lc3_learning_the_shell.php)
+The bash manual is available to you in most bash shells, although not in Git Bash. If you are using Git Bash, an internet search for "`command_name` bash manual" will usually get you the same information.
 
-- Software Carpentries offers an open source course [introducing new users to the Unix Shell](https://swcarpentry.github.io/shell-novice/)
-
-### The bash manual
+</div>
 
-The bash manual is available to you whenever you are in a bash shell.  Entering `man command_name` reveals the documentation (`man` is short for "manual") for the function `command_name`.  This can be useful if you think a command might be useful and want to learn more about it, or have forgotten how to use a command or what its options are.
+Entering `man command_name` reveals the documentation (`man` is short for "manual") for the function `command_name`.  This can be useful if you think a command might be useful and want to learn more about it, or have forgotten how to use a command or what its options are.
```

Comment:
```suggestion
Entering `man command_name` reveals the documentation for the function `command_name`  (`man` is short for "manual").  This can be helpful if you think a command might be useful and want to learn more about it, or have forgotten how to use a command or what its options are.
```
1. Improve sentence flow
2. Avoids using useful 2x in one sentence